docs/usage/self-hosting.md

```diff
@@ -2,43 +2,45 @@
 
 ## Installing Renovate OSS CLI
 
-#### npmjs
+### npmjs
 
 ```sh
 $ npm install -g renovate
 ```
 
-Since renovate v20 `npm`, `pnpm` and `yarn` are no longer embedded, so you need to install them globally if you need to update lockfiles.
+Starting from version 20, Renovate does not embed `npm`, `pnpm` and `yarn` anymore.
+If you want to use these package managers to update your lockfiles, you must install them globally.
 
 ```sh
 $ npm install -g yarn pnpm
 ```
 
```

Comment:
Why are we including this example command?

```sh
 $npm install -g yarn pnpm
```

First we say: Starting from version 20, Renovate does not embed `npm`, `pnpm` and `yarn` anymore.
And then we show a example of which uses `npm` to instal the `yarn` and `pnpm` managers...

I must be confused or missing something for this to make sense. 🙃

Reply:
Well, node comes with `npm` pre-installed. So we mean we don't ship an up-to-date version of npm.